### PR TITLE
Add --integration-versions-including argument to release tool.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: None

Signed-off-by: Kristian Amlie <kristian.amlie@mender.io>

### CODE REVIEW

File: extra/release_tool.py [hunk 1]
```diff
@@ -173,14 +175,21 @@ def docker_compose_files_list(dir):
...
+    data = {}
+    for json_str in json_list:
+        deepupdate(data, yaml.load(json_str))
+    return data
+
 def get_docker_compose_data(dir):
     """Return the Yaml as data from all the docker-compose YAML files."""
-    data = {}
+    json_list = []
     for file in docker_compose_files_list(dir):
```
Comment (GregorioDiStefano) [inline extra/release_tool.py]: 'file' is a built in, not sure if it's good idea to use it as a variable

Comment (kacf) [inline extra/release_tool.py]: The linter didn't complain, and I've been using this a lot. What is it?

File: extra/release_tool.py [hunk 2]
```diff
@@ -955,14 +967,67 @@ def do_set_version_to(args):
...
+        candidates.append(line)
+
+    # Now look at each docker compose file in each branch, and figure out which
+    # ones contain the version of the service we are querying.
+    matches = []
+    for candidate in candidates:
+        yamls = []
+        files = execute_git(None, git_dir, ["ls-tree", "--name-only", candidate],
+                            capture=True).strip().split('\n')
+        for file in files:
```
Comment (GregorioDiStefano) [inline extra/release_tool.py]: same here

File: extra/release_tool.py (hunk 1 above)
Comment (GregorioDiStefano) [inline extra/release_tool.py]: http://stackoverflow.com/a/2418007

Comment (kacf) [inline extra/release_tool.py]: Both fixed.

**Pull request merged**